src/connect/mapDispatchToProps.ts

```diff
@@ -1,21 +1,28 @@
+import { ActionCreator, ActionCreatorsMapObject, Dispatch } from 'redux'
 import bindActionCreators from '../utils/bindActionCreators'
 import { wrapMapToPropsConstant, wrapMapToPropsFunc } from './wrapMapToProps'
 
-export function whenMapDispatchToPropsIsFunction(mapDispatchToProps) {
+export function whenMapDispatchToPropsIsFunction(
+  mapDispatchToProps: ActionCreatorsMapObject | ActionCreator<any>
+) {
   return typeof mapDispatchToProps === 'function'
     ? wrapMapToPropsFunc(mapDispatchToProps, 'mapDispatchToProps')
     : undefined
 }
 
-export function whenMapDispatchToPropsIsMissing(mapDispatchToProps) {
+export function whenMapDispatchToPropsIsMissing(
+  mapDispatchToProps: ActionCreatorsMapObject | ActionCreator<any>
```

Comment:
:hand: Here, it's probably just `undefined`.

Reply:
Shoul I let the `!mapDispatchToProps` check ?